ios/browser/api/ai_chat/ai_chat.h

```diff
@@ -35,7 +43,8 @@ OBJC_EXPORT
 
 - (void)changeModel:(NSString*)modelKey;
 
-- (void)submitHumanConversationEntry:(NSString*)text;
+- (void)submitHumanConversationEntry:(NSString*)text
+                              images:(nullable NSArray<UIImage*>*)images;
```

Comment:
Should these changes also move to the next branch?

Reply:
I didn't want to move them, but it's moved now.